taskcluster/ci/extract-best/kind.yml

```diff
@@ -17,8 +18,8 @@ kind-dependencies:
     - translate-corpus
 
 tasks:
-    "{src_locale}-{trg_locale}-{this_chunk}/{total_chunks}":
-        description: extract best {src_locale}-{trg_locale} {this_chunk}/{total_chunks}
+    "{src_locale}-{trg_locale}-{{this_chunk}}/{{total_chunks}}":
```

Comment:
This might be kind of weird to someone reading the code but I can't think of anything besides escaping the `{ }`.

Adding a comment might be helpful.

Reply:
Yeah, it's not ideal. This is actually why [we allow partial substitutions](https://github.com/mozilla/firefox-translations-training/blob/bf7e1b1d080772efb5abdaffad26a2a3e8327146/taskcluster/translations_taskgraph/util/substitution.py#L1) for the remaining transforms here. I don't think that's something we actually want upstream though...

I'll add a comment.